### PR TITLE
add mocha, chai, sinon, enzyme testing stack, FileList unit tests

### PR DESCRIPTION
**Pull request by avahowell**
This PR adds the necessary dependencies for testing the UI.  Included is the testing for the FileList component of the renter, more tests will follow in later PRs.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015", "react"]
+}
```
Comment (lukechampine) [inline .babelrc:3]: damn, I thought we could get away with not having a babelrc

**Comment by lukechampine** [comment]
From Travis:

```
Warning: Failed propType: Required prop `showSearchField` was not specified in `FileList`.
```

Should we bother correcting these?

**Comment by avahowell** [comment]
Yeah, I think a required prop should always be passed even in testing.

File: .babelrc (hunk 1 above)
Comment (avahowell) [inline .babelrc:3]: `--compilers js:babel-register` :/

**Pull request merged**